libs/openFrameworks/gl/ofFbo.cpp

```diff
@@ -303,10 +303,10 @@ ofFbo::ofFbo(const ofFbo & mom){
 }
 
 //--------------------------------------------------------------
-void ofFbo::operator+=(const ofFbo & f){
-	if (f.isAllocated()) {
+void ofFbo::operator+=(const ofFbo & fbo){
+	if (fbo.isAllocated()) {
 		this->begin();
-		f.draw(0,0);
+		fbo.draw(0,0);
 		this->end();
 	}
```

Comment:
Feel like it should be something like: 

```
	//if we aren't allocated lets allocate to be the same as the input
	if (!this->isAllocated() && fbo.isAllocated()) {
 		this->allocate(fbo.settings); 
 	}
	if (fbo.isAllocated()) {
 		this->begin();
 		fbo.draw(0,0);
 		this->end();
 	}else{
 		ofLogWarning("ofFbo::operator+=") << " input fbo is not allocated, skipping += ";
 	}
```

Reply:
@dimitre what do you think about the suggestion above?👆

This way it basically copies the fbo being += if the main fbo isn't allocated